daemon/container.go

```diff
@@ -705,9 +711,12 @@ func (container *Container) Stop(seconds int) error {
 	// 1. Send a SIGTERM
 	if err := container.KillSig(15); err != nil {
 		log.Infof("Failed to send SIGTERM to the process, force killing")
-		if err := container.KillSig(9); err != nil {
-			return err
+		if err != syscall.ESRCH {
+			if err := container.KillSig(9); err != nil && err != syscall.ESRCH {
```

Comment:
Oops, sorry. The message would be duplicated if the process terminates before `kill 9` and after `kill 15` and I still don't see a benefit of it.

Just to be sure, are we talking about this:

	if err := container.KillSig(15); err != nil {
 		log.Infof("Failed to send SIGTERM to the process, force killing")
		if err != syscall.ESRCH {
			if err := container.KillSig(9); err != nil {
				if err != syscall.ESRCH {
					return err
				}
				log.Debugf("Cannot kill process (pid=%d): no such process.", container.GetPid())
			}
 		}
		log.Debugf("Cannot kill process (pid=%d): no such process.", container.GetPid())
 	}

?
The second debug statement could be put into an else branch which would prevent the duplication. But still the outcome would be the same as with current code.